### PR TITLE
docs(dip27): update withdrawal safety to match v22

### PR DESCRIPTION
**Pull request by thephez**
Dash Core v22 makes simplifies withdrawal safety rules and also makes them less restrictive as implemented in dashpay/dash#6369.

### CODE REVIEW

File: dip-0027.md [hunk 1]
```diff
@@ -142,6 +142,10 @@ Asset Unlock transactions might not be mined for multiple reasons. For example,
 As a way to avoid a catastrophic failure if Platform is compromised, Core will limit credit pool withdrawals for at least the first release of Platform. The withdrawal limits may be re-evaluated and updated periodically as the system matures. To evaluate withdrawal validity, withdrawal amounts from the last 576 blocks should be tallied. The withdrawal should not be mined if:
 
 * It requests more DASH than the credit pool contains
+* The withdrawal would result in more than a 2000 DASH reduction in the credit pool over the 576 block-window
```
Comment (PastaPastaPasta) [inline dip-0027.md:145]: `over the past 576 blocks` instead?

Comment (knst) [reply dip-0027.md:145]: > over the past 576 blocks instead?

that's same wording as below, if so maybe should update both lines:

> * The withdrawal would result in more than a 1000 DASH reduction in the credit pool over the 576 block-window

Comment (thephez) [reply dip-0027.md]: Updated in both places

**Comment by thephez** [comment]
@PastaPastaPasta @knst please re-review

**Review by knst — APPROVED** [review]
LGTM 013d97086391e51c21bf0a2b1471f0e3f825d32c

**Comment by thephez** [comment]
@PastaPastaPasta re-review please :pray:

**Review by PastaPastaPasta — APPROVED** [review]
LGTM

**Pull request merged**